src/index.ts

```diff
@@ -647,9 +832,29 @@ function readConfig (
   }
 
   // Override default configuration options `ts-node` requires.
-  config.compilerOptions = Object.assign({}, config.compilerOptions, options.compilerOptions, TS_NODE_COMPILER_OPTIONS)
+  config.compilerOptions = Object.assign(
+    {},
+    config.compilerOptions,
+    config['ts-node']?.compilerOptions,
+    options.compilerOptions,
+    TS_NODE_COMPILER_OPTIONS
+  )
+
+  const fixedConfig = fixConfig(ts, ts.parseJsonConfigFileContent(config, ts.sys, basePath, undefined, configFileName))
+
+  // Fix ts-node options that come from tsconfig.json
+  const tsconfigOptions: TsConfigOptions = {
+    ...config['ts-node']
+  }
+  if (tsconfigOptions.requires) {
```

Comment:
If we want this we should do it as part of a follow up PR.

Reply:
Referring specifically to specifying `requires` in a tsconfig file?